task 2/2 intro task.playground/Contents.swift

```diff
@@ -0,0 +1,142 @@
+public class Stack {
+    fileprivate var stackElements = [Int]()
+    
+    init() {}
+    init(_ initStackElements: [Int]){
+        self.stackElements = initStackElements
+      }
+
+    /**
+    * Adds element on the top of the stack
```

Comment:
Please pay more attention:

Probably you do not want to add `*` before the description:
```suggestion
     Adds element on the top of the stack
```

It will render the Summary section.

Reply:
Here and below.

Reply:
Thank you, I will change it.